src/main/java/seedu/address/logic/commands/scorecommand/AddScoreCommand.java

```diff
@@ -0,0 +1,61 @@
+package seedu.address.logic.commands.scorecommand;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_TEAM_DISPLAYED_INDEX;
+
+import seedu.address.commons.exceptions.AlfredException;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.entity.Id;
+import seedu.address.model.entity.PrefixType;
+import seedu.address.model.entity.Score;
+import seedu.address.model.entity.Team;
+
+/**
+ * Command for adding a certain score to a team's current score.
+ */
+public class AddScoreCommand extends ScoreCommand {
+
+    public static final String MESSAGE_SCORE_TEAM_SUCCESS = "Added %1$s points to %2$s's score";
+    public static final String COMMAND_WORD = "score add";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": adds the specified score to specified team's current score. "
+            + "If the team's new score exceeds " + Score.MAX_SCORE + " it will be set to " + Score.MAX_SCORE + "\n"
+            + "Format: " + COMMAND_WORD + " [teamID] score \n"
+            + "For example: " + COMMAND_WORD + " T-5 25";
+
+    public AddScoreCommand(Id teamId, Score score) {
+        super(teamId, score);
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+        Team teamToScore;
+
+        try {
+            teamToScore = model.getTeam(id);
+        } catch (AlfredException ae) {
+            throw new CommandException(MESSAGE_INVALID_TEAM_DISPLAYED_INDEX);
+        }
+
+        try {
+            model.addTeamScore(teamToScore, score);
+        } catch (AlfredException e) {
```

Comment:
this is a little nitpicky, but AlfredException is given the variable e here, but is given the variable ae above. LOL

Reply:
sure i'll change it